blivet/formats/disklabel.py

```diff
@@ -223,8 +223,12 @@ def get_platform_label_types(cls):
         label_types = ["msdos", "gpt"]
         if arch.is_pmac():
             label_types = ["mac"]
+        elif arch.is_aarch64():
+            label_types = ["gpt", "msdos"]
         elif arch.is_efi() and not arch.is_aarch64():
             label_types = ["gpt"]
+        elif arch.is_efi() and arch.is_arm():
```

Comment:
this section should be moved above, as  it has efi and the arch is not aarch64, we should never get this far

Reply:
Good catch